src/main/java/seedu/address/logic/commands/add/AddCommand.java

```diff
@@ -158,7 +158,8 @@ public CommandResult execute(Model model) throws CommandException {
             }
 
             model.addConsultation(toAddConsultation);
-            return new CommandResult(String.format(MESSAGE_SUCCESS_CONSULTATION, toAddConsultation));
+            return new CommandResult(String.format(MESSAGE_SUCCESS_CONSULTATION, toAddConsultation),
+                    CommandTargetFeature.Consultations);
```

Comment:
thanks for implementing switching of tabs to consultation for add consultation command